suggesters/simple_identification_suggester.py

```diff
@@ -0,0 +1,110 @@
+from typing import List, Tuple, Dict
+from suggesters.protocols import ModelerProtocol
+import networkx as nx
+import guidance
+from enum import Enum
+import re
+import itertools
+
+class SimpleIdentificationSuggester:
+
+    def suggest_backdoor(
```

Comment:
a docstring in the sphinx format (see dowhy code for examples) will be useful for each method.